readme.md

```diff
@@ -230,6 +279,13 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
-```
+var andy = Instructor.find({'name': 'Andy'}, function(err, users){
+  if (err) throw err;
+});
+
+var andrew = andy.Wishlist_Item.create({description: 'Resin Laying Deer Figurine, Gold'});
+andy.save(function(err){
+  if (err) return handleError(err);
+})
 
```

Comment:
Hmm, i'm not entirely sure what this will do because of the async nature of the queries. Nesting things within your find query will ensure things happen when you want them too. Like this:

```js
Instructor.findOne({name: "Andy"), function(err, docs){
  docs.wishlist_items.push({description: "Resin Laying Deer Figurine, Gold");
  docs.save();
});
```